sdk/nodejs/location.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export let EastAsia:           Location = "eastasia";
```

Comment:
Do we want to consider putting these under an `azure.location` namespace? I think given the names keeping them at the top level is also fine though.

Reply:
discussed offline and lets leave them at the top level for now